hugo/content/faq/citing.md

```diff
@@ -1,10 +0,0 @@
----
-Title: How do I cite publications in the ACL Anthology?
-weight: 1
----
-
-If available, you can you use Digital Object Identifiers (DOIs).
-The canonical URL for papers is given by appending the ACL ID to the Anthology URL, e.g., https://www.aclweb.org/anthology/E91-1001
-Both the DOIs and the persistent URLs embed the 8-character ACL Anthology Identifier.
-
-You can use variations on the canonical URL to access [the paper page](https://www.aclweb.org/anthology/E91-1001) (by appending a `/`) or [the BibTeX file](https://www.aclweb.org/anthology/E91-1001.bib) (by appending `.bib`).
```

Comment:
Any particular reason this was deleted?

Reply:
I think I got thrown off by the word “cite”—you cite papers by using the BibTeX or other formats we export! But now I think the question is probably one about the canonical URLs. I’ll rewrite this in terms that make this more clear and add it back in.

Reply:
Okay, I rewrote this page.